### PR TITLE
[ZH] Add `Lazer/Feature comparison`

### PR DESCRIPTION
**Pull request by Sunnyboy971**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+| 有意更改 | 是 |
+| 需要进一步考虑 | 否 |
+
+#### 未点击时，滑条头沿路径移动
+
+![](img/moving-slider-head.gif)
+
+当滑条达到开始时间时，滑条头会开始沿路径移动，同时会移动实际打击目标。与之相比，稳定版中的打击目标留在原位。
+
+当你看到滑条的时候，这很有道理。但这也增加了计算游玩的复杂度（也潜在地增加了回放中的错误，尤其是 OD 值与滑条速度较高时）。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:162]:
```suggestion
玩家在看到滑条后能理解怎么去击打。但这也增加了计算游玩的复杂度（也潜在地增加了回放中的错误，尤其是判定严度 (OD) 与滑条速度 (SV) 较高时）。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+
+此举背后的理由很简单：不这样做，启用`渐出滑条`时，如果玩家点击滑条头迟了，就会很难看出来。你会让圆圈停在原地，延迟渐出过程吗？如果这样做，圆圈该怎么跟上滑条球的实际位置呢？相反，如果不延迟渐出过程，你会让滑条头圆圈与滑条本身分开吗（请别这么做）？
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:164]:
```suggestion
这个改变背后的理由很简单：如果不这样做，当玩家较晚点击滑条头时，启用`渐出滑条`设置时，从视觉上来考虑，会很麻烦。比如，较晚点击时，在滑条头原处留个圆圈，推迟滑条头缩回的时机？如果按照这样做，那么滑条头该怎么样跟上滑条球的实际位置？除此之外，难道还能暂不缩回滑条，让滑条头与滑条主体分离吗？不要考虑太多。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 是 |
+| 有意更改 | 是 |
+| 需要进一步考虑 | 是 |
+
+#### 滑条尾不会增加连击数，也不会导致 miss
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:173]:
```suggestion
#### 滑条尾不会计入连击数，也不会导致失误 (miss)
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 4]
```diff
@@ -0,0 +1,444 @@
...
+
+滑条尾的行为一度很怪异，漏掉它不会导致断连（也不会导致 MISS 判定）。这意味着 S 评价的游玩（需要无 MISS 判定）尽管漏了几个滑条尾，也看起来像是全连了。关于是否应称此类游玩为全连的争论随之而来。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:175]:
```suggestion
滑条尾的行为一度很怪异，漏掉它不会导致断连（也不会导致 MISS 判定）。这意味着 S 评价的游玩（需要无 MISS 判定）尽管漏了几个滑条尾，也看起来像是全连了。随之而来的是大量的关于是否应该将这种成绩纳入全连的争论。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 5]
```diff
@@ -0,0 +1,444 @@
...
+从现在开始，滑条尾只会影响分数和准确度，但不会影响连击数。这意味着 S 评价总代表全连。
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 是 |
+| 有意更改 | 是 |
+| 需要进一步考虑 | 是 |
+
+#### 漏掉滑条头会导致 miss
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:186]:
```suggestion
#### 漏掉滑条头会导致失误 (miss)
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 6]
```diff
@@ -0,0 +1,444 @@
...
+
+之前，漏掉滑条头（没有点击，或是在 miss 窗口期内点击）会断连，但不会导致 MISS 判定。如果完成滑条剩下的部分，仍然可以获得判定。这让玩家在最大连击数较低的情况下仍可获得 S 评价。
+
+在 lazer 中，不点击滑条头会对整个滑条给出 MISS 判定。在此之后，仍然可以从滑条点与折返中获得连击数、分数与准确度，但从滑条尾只能获得分数与准确度。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:190]:
```suggestion
在 lazer 中，错过滑条头，整个滑条会获得 MISS 判定。在此之后，你仍然可以从滑条点与折返中获得连击数、分数与准确度，从滑条尾上获得分数与准确度。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 7]
```diff
@@ -0,0 +1,444 @@
...
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 是 |
+| 需要进一步考虑 | 否 |
+
+#### 滑条尾未被打击时不播放音效
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:199]:
```suggestion
#### 漏掉滑条尾时不播放音效
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 8]
```diff
@@ -0,0 +1,444 @@
...
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 否 |
+| 需要进一步考虑 | 是 |
+
+### osu!taiko
+
+#### 无法击打与 swell 重叠的音符
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:238]:
```suggestion
#### 无法击打与转盘覆盖区域重叠的音符
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 9]
```diff
@@ -0,0 +1,444 @@
...
+
+一些机关图利用了与 swell 重叠的音符。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:240]:
```suggestion
一些技巧 (gimmick) 图含有与转盘覆盖区域重叠的音符。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 10]
```diff
@@ -0,0 +1,444 @@
...
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 否 |
+| 需要进一步考虑 | 是 |
+
+#### 不会阻止乱打滑条
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:249]:
```suggestion
#### 不需要按照鼓点击打黄条
```

这里需要改一下翻译，原文很难直译

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 11]
```diff
@@ -0,0 +1,444 @@
...
+
+在稳定版中，滑条不能击打得太快，也不能太慢。像在 ScoreV2 模组中一样，已经去除了这样的限制。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:251]:
```suggestion
在稳定版中，黄条不能击打得太快或太慢。像在 ScoreV2 模组中一样，lazer 已经去除了这样的限制。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 12]
```diff
@@ -0,0 +1,444 @@
...
+| 破坏向后兼容性 | 否 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 否 |
+| 需要进一步考虑 | 是 |
+
+### osu!catch
+
+#### 某些情况下，红果跳生成方式可能不同
+
+这可能会导致回放中的判定不准与难度增加。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:277]:
```suggestion
这可能会导致回放中的判定不准，并且会增加实际难度。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 13]
```diff
@@ -0,0 +1,444 @@
...
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 否 |
+| 需要进一步考虑 | 是 |
+
+### osu!mania
+
+#### 会对长按音符的头和尾进行判定
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:299]:
```suggestion
#### 长按音符的头尾都有判定
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 14]
```diff
@@ -0,0 +1,444 @@
...
+这与稳定版中的 ScoreV2 模组功能相似。
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 是 |
+| 需要进一步考虑 | 否 |
+
+#### 长按音符点得分，也会导致断连
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:310]:
```suggestion
#### 长按音符中的点会提供分数或者导致断连
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 15]
```diff
@@ -0,0 +1,444 @@
...
+
+在稳定版中，在长按音符体中松开按键会立刻导致断连，但现在只会在漏掉音符点时断连。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:312]:
```suggestion
在稳定版中，按住长按音符后，松开按键会立刻导致断连。但现在只会在漏掉长按音符中的点时断连。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 16]
```diff
@@ -0,0 +1,444 @@
...
+
+当需要用另一个手指按住按键时，这点更改允许使用手腕，从而允许 cheesing （即 “manipulating”）。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:314]:
```suggestion
这允许玩家在一根手指按住按键，同时需要用另一根手指点击时，通过手腕的发力来提起手指并同时按下两个按键 (cheesing，或者 manipulating)。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 17]
```diff
@@ -0,0 +1,444 @@
...
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 是 |
+| 需要进一步考虑 | 是 |
+
+#### 限制极限滚动速度
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:323]:
```suggestion
#### 限制极限滚动速度 (SV)
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 18]
```diff
@@ -0,0 +1,444 @@
...
+
+<!-- TODO: how exactly -->
+
+一些含有 SV 机关（比如物件传送与停滞）的谱面可能看起来与预期不符，但除此之外都是可玩的。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:327]:
```suggestion
一些含有变速梗（比如物件瞬移与停滞）的谱面可能看起来与预期不符，但除此之外的谱面都是可玩的。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 19]
```diff
@@ -0,0 +1,444 @@
...
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 否 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 是 |
+| 需要进一步考虑 | 是 |
+
+#### PERFECT 判定窗口期与 OD 值成正比
+
+无论整体难度如何，这个窗口期过去恒为 ±16 毫秒。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:338]:
```suggestion
无论整体难度 (OD) 设为多少，之前这个判定固定为 ±16 毫秒。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 20]
```diff
@@ -0,0 +1,444 @@
...
+在稳定版中， osu!mania 恢复了使用 DT/HT 模组对打击窗口的影响。
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 否 |
+| 需要进一步考虑 | 是 |
+
+#### Flashlight 模组没有梯度
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:369]:
```suggestion
#### Flashlight 模组可视区域周围没有渐变
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 21]
```diff
@@ -0,0 +1,444 @@
...
+<!-- TODO: comparison image  -->
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 否 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 否 |
+| 需要进一步考虑 | 是 |
+
+### 分数、表现分和排行榜
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:380]:
```suggestion
### 分数、表现分 (pp) 和排行榜
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 22]
```diff
@@ -0,0 +1,444 @@
...
+
+lazer 的计分系统被简化了，主要好在无需回放来重新计算得分。增加了两套可互换的系统：*标准化*和*经典*。标准化计分将得分限制为最高 1,000,000 分 + 附加分 + 得分乘数，而经典计分类似于标准化，但与谱面中的物件数平方成正比。可以在设置中进行选择，游戏内的分数显示区域也会相应改变。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:382]:
```suggestion
lazer 中使用了简化过的计分系统。这套系统的主要优点是不需要播放一遍回放后才能计算分数。这套计分系统有两个可更换的计分方式：标准和经典。标准计分则将基础分数限制为 1000000 分乘以分数倍率 + 奖励分。经典计分和标准类似，但是它的分数是根据谱面中已经击打物件的数量来平方缩放过的。这两个计分方式可以在设置中切换，并且会影响游戏中所有分数显示。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 23]
```diff
@@ -0,0 +1,444 @@
...
+
+在每个打击物件与判定对应的分数上也存在一些差异。
+
+依赖于分数的功能，如[分数排名](/wiki/Ranking#score-ranking)和[等级](/wiki/Gameplay/Score/Total_score#level)受到的影响需要进一步讨论。
+
+不会颁发奖牌。这个功能会在实现防作弊措施后被启用，也要进一步考虑由于其他游玩方面的改变导致的难度变化。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:388]:
```suggestion
lazer 中不会得到奖章（成就）。将来会在防作弊实装后启用这个功能，并且，我们也要进一步考虑由于其他游玩方面的改变，可能导致的难度变化。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 24]
```diff
@@ -0,0 +1,444 @@
...
+
+表现分现在还没整合到全球或特定谱面的排行榜中，而是整合到了单独版本的网站中： <https://lazer.ppy.sh/>。
+
+### 多人游戏
+
+#### 未实现 TAG co-op 模式
+
+已经计划加入此功能，但还没有开始在其上工作。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:396]:
```suggestion
已经计划加入此功能，但还没有开始在这个项目中工作。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 25]
```diff
@@ -0,0 +1,444 @@
...
+
+![](img/sb-crop.jpg)
+
+某些故事板被设计为只支持 4:3 宽高比，但在 lazer 中，总按 16:9 比例渲染故事板。
+
+## 缺失的外观功能
+
+计划在某个时候重新加入下列功能，但由于优先级较低暂未实现。
+
+- [Niconico](https://en.wikipedia.org/wiki/Niconico) 式的滚动回放评论
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:426]:
```suggestion
- [Niconico](https://en.wikipedia.org/wiki/Niconico) 式的滚动回放评论（弹幕）
```

**Review by YumeMuzi — CHANGES_REQUESTED** [review]
There are many long and difficult sentences in the article that need to be carefully translated.

**Comment by Sunnyboy971** [comment]
Thank you for your careful review! ❤️

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 26]
```diff
@@ -0,0 +1,444 @@
...
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 是 |
+| 需要进一步考虑 | 是 |
+
+#### 不支持 Aspire 样式的故障滑条
+
+![](img/aspire-slider.gif)
+
+一些冒险性质的谱面利用了稳定版客户端中允许使用不寻常滑条机制的漏洞，小到作为不可见圆圈使用的零长度滑条，大到跨屏幕拉伸与压缩的滑条。
```
Comment (YumeMuzi) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:225]:
```suggestion
一些 Aspire 谱面利用了稳定版客户端中允许使用不寻常滑条机制的漏洞，小到作为不可见圆圈使用的零长度滑条，大到跨屏幕拉伸与压缩的滑条。
```
这里忘记提了 aspire 国内暂且没有翻译，就不直译了

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 27]
```diff
@@ -0,0 +1,444 @@
+# osu!(lazer) 功能比较
+
+*对于更简略的功能列表，参见： [帮助中心/升级到 lazer § 功能对比](/wiki/Help_centre/Upgrading_to_lazer#功能对比)*
+
+此页记录了与 osu!(稳定版) 相比，所有新增与缺失的功能，以及影响游玩的差异之处。
+
+## 新功能
+
+- 跨平台支持
+  - 对 Windows 8.1+、 macOS、 Linux、 iOS 与 Android 的本地支持
```
Comment (kj415j45) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:10]:
```suggestion
  - 对 Windows 8.1+、 macOS、 Linux、 iOS 与 Android 的原生支持
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 28]
```diff
@@ -0,0 +1,444 @@
...
+在 lazer 之前，滑条头只需 50/MEH 准确度判定，即可获得完美得分。这么做是出于历史原因，但对于音游来说，感觉很不好。之后，玩家需要在精准的时机点击滑条头。
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 是 |
+| 有意更改 | 是 |
+| 需要进一步考虑 | 否 |
+
+#### 即使不点击，滑条头也会沿路径缩回
```
Comment (kj415j45) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:156]:
```suggestion
#### 即使不点击，滑条头也会沿路径移动
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 29]
```diff
@@ -0,0 +1,444 @@
...
+
+![](img/moving-slider-head.gif)
+
+当滑条达到开始时间时，滑条头会开始沿路径移动，实际打击目标也会跟着移动。与之相比，稳定版中的打击目标视觉上却留在原位。
+
+玩家在看到滑条后能理解怎么去击打。但这也增加了计算游玩的复杂度（也潜在地增加了回放中的错误，尤其是判定严度 (OD) 与滑条速度 (SV) 较高时）。
+
+这个改变背后的理由很简单：如果不这样做，当玩家较晚点击滑条头时，启用`渐出滑条`设置时，从视觉上来考虑，会很麻烦。比如，较晚点击时，在滑条头原处留个圆圈，推迟滑条头缩回的时机？如果按照这样做，那么滑条头该怎么样跟上滑条球的实际位置？除此之外，难道还能暂不缩回滑条，让滑条头与滑条主体分离吗？不要考虑太多。
```
Comment (kj415j45) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:164]:
```suggestion
这个改变背后的理由很简单：如果不这样做，启用`渐出滑条`设置时，一旦玩家操作滞后就会难以辨识此时应击打的位置：假如在滑条头初始位置留下圆圈，延后滑条渐出，此时该如何让滑条头跟上实际的滑条球位置呢？换句话说，假如不延后滑条渐出，此时让滑条头与滑条球分离会更好吗？
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 30]
```diff
@@ -0,0 +1,444 @@
...
+| 有意更改 | 是 |
+| 需要进一步考虑 | 是 |
+
+#### 不支持 Aspire 样式的故障滑条
+
+![](img/aspire-slider.gif)
+
+一些 Aspire 谱面利用了稳定版客户端中允许使用不寻常滑条机制的漏洞，小到作为不可见圆圈使用的零长度滑条，大到跨屏幕拉伸与压缩的滑条。
+
+对于对 Aspire 谱面的兼容性，还需要进一步讨论与考虑。比如，不可见圆圈未来可能成为一个得到适当支持的特性。
```
Comment (kj415j45) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:227]:
```suggestion
对于对 Aspire 谱面的兼容性，还需要进一步讨论与考虑。比如，不可见圆圈未来可能成为一个得到正确支持的特性。
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 31]
```diff
@@ -0,0 +1,444 @@
...
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 否 |
+| 需要进一步考虑 | 是 |
+
+### osu!mania
+
+#### 长按音符的头尾都有判定
```
Comment (kj415j45) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:299]:
```suggestion
#### 长按物件的头尾都有判定
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 32]
```diff
@@ -0,0 +1,444 @@
...
+这与稳定版中的 ScoreV2 模组功能相似。
+
+|  |  |
+| :-- | :-: |
+| 破坏向后兼容性 | 是 |
+| Classic 模组的恢复支持 | 否 |
+| 有意更改 | 是 |
+| 需要进一步考虑 | 否 |
+
+#### 长按音符中的点会提供分数或者导致断连
```
Comment (kj415j45) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:310]:
```suggestion
#### 长按物件中的点会提供分数或者导致断连
```

File: wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md [hunk 33]
```diff
@@ -0,0 +1,444 @@
...
+| 有意更改 | 是 |
+| 需要进一步考虑 | 否 |
+
+#### 即使不点击，滑条头也会沿路径缩回
+
+![](img/moving-slider-head.gif)
+
+当滑条达到开始时间时，滑条头会开始沿路径移动，实际打击目标也会跟着移动。与之相比，稳定版中的打击目标视觉上却留在原位。
+
+玩家在看到滑条后能理解怎么去击打。但这也增加了计算游玩的复杂度（也潜在地增加了回放中的错误，尤其是判定严度 (OD) 与滑条速度 (SV) 较高时）。
```
Comment (kj415j45) [inline wiki/Client/Release_stream/Lazer/Feature_comparison/zh.md:162]:
```suggestion
玩家在看到滑条后能理解怎么去击打。但这也增加了计算游玩的复杂度（也潜在地增加了回放中的错误，尤其是整体难度 (OD) 与滑条速度 (SV) 较高时）。
```

**Review by kj415j45 — APPROVED** [review]
LGTM

**Review by TicClick — APPROVED** [review]

**Pull request merged**